src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -29,7 +29,7 @@ public enum CustomQueryInterfaceMode
     public static partial class Marshal
     {
 #if FEATURE_COMINTEROP
```

Comment:
It would be nice to keep the textual Guid as a comment to make it possible to find via full text search